LWA/LWA_bifrost.py

```diff
@@ -815,12 +815,15 @@ def __init__(
         self.ntime_gulp = ntime_gulp
         self.accumulation_time = accumulation_time
 
-        self.antennas = antennas
+        self.station = station
         locations = numpy.empty(shape=(0, 3))
-        for ant in self.antennas:
+        for ant in self.station.antennas:
             locations = numpy.vstack((locations, [ant.stand[0], ant.stand[1], ant.stand[2]]))
         locations = numpy.delete(locations, list(range(0, locations.shape[0], 2)), axis=0)
-        locations[255, :] = 0.0
+        if self.station == lwasv:
+            locations[[i for i, a in enumerate(self.station.antennas[::2]) if a.stand.id == 256], :] = 0.0
+        elif self.station == lwa1:
+            locations[[i for i, a in enumerate(self.station.antennas[::2]) if a.stand.id in (35, 257, 258, 259, 260)], :] = 0.0
```

Comment:
I'm a little concerned about these lines. Never really noticed them before. are we just setting the location of all outriggers to 0?

Reply:
Isn't this how it had always dealt with the outriggers?  I think this is "safe" since it is coupled with a weight of zero later on.  What happens if we do try to grid the outriggers?

Reply:
yeah I had all the same thoughts!